frontend/packages/metal3-plugin/src/components/baremetal-hosts/dashboard/BareMetalHostDashboard.tsx

```diff
@@ -14,9 +16,11 @@ const BareMetalHostDashboard: React.FC<BareMetalHostDashboardProps> = ({
   machines,
   nodes,
 }) => {
+  const machine = machines.find((m) => getName(m) === getHostMachineName(obj));
```

Comment:
there is a selector for this:
```suggestion
  const machine = getHostMachine(obj, machines);
```

Reply:
Done!